lib/fluent/plugin/in_rds_mysql_log.rb

```diff
@@ -1,7 +1,18 @@
+require 'fluent:input'
```

Comment:
Sorry it should actually be require 'fluent/input'   Its needed that way for the latest version of fluentd.  I actually duplicated your plugin and made the changes i need and am adding more to meet the needs of my company.  So, i no longer need these changes here, if you do not want to merge the PR.  I pushed the changes up to ruby gems and plan to make quite a few more in the plugin itself. Also, added travis ci and rake testing which will be deployed in the next couple of days. I gave you credit for the original code.  great work. Again, if you want to close this PR please fee free.